src/mn_setup.c

```diff
@@ -3239,6 +3240,10 @@ static const char *exit_sequence_strings[] = {
     "Off", "Sound Only", "PWAD ENDOOM", "Full"
 };
 
+static const char *fuzzmode_strings[] = {
+    "Blocky", "Selective", "Selective Dark"
```

Comment:
Temporary names. Any suggestions?

Reply:
Not sure if we actually need "selective" at all. It's really hard to see, even worse than "original". How about "original" and "alternative" instead of "selective dark"?

Reply:
I've made it slightly darker there: https://github.com/fabiangreffrath/woof/pull/2043/commits/11a95d9fea49387d29e6fdb5624eb7b3cbf17705

I didn't get a good result with translucency + “selective”, maybe @ceski-1 can help.

Reply:
There is still a nomenclature mixture of "selective" and "refraction".

Reply:
> There is still a nomenclature mixture of "selective" and "refraction".

Fixed in 2e7f028